test/unit/lib/plugins/aws/package/compile/layers.test.js

```diff
@@ -343,18 +60,36 @@ describe('lib/plugins/aws/package/compile/layers/index.test.js', () => {
       awsRequestStubMap,
     });
     cfResources = cfTemplate.Resources;
+    cfOutputs = cfTemplate.Outputs;
     naming = awsNaming;
+    service = serverless.service;
     ({ updateConfig, servicePath } = fixtureData);
   });
 
-  it.skip('TODO: should support `layers[].package.artifact` with `package.individually`', () => {
-    // Replaces
-    // https://github.com/serverless/serverless/blob/e78f695004bf292c4163daf9705e5e0c6cbe2592/test/unit/lib/plugins/aws/package/compile/layers/index.test.js#L90-L106
+  it('should support `layers[].package.artifact` with `package.individually`', () => {
+    const resourceName = 'layer';
+    const layerResource = cfResources[naming.getLambdaLayerLogicalId(resourceName)];
+    const s3Folder = service.package.artifactDirectoryName;
+    const s3FileName = service.layers[resourceName].package.artifact.split(path.sep).pop();
+
+    expect(layerResource.Properties.Content.S3Key).to.be.equal(`${s3Folder}/${s3FileName}`);
   });
 
-  it.skip('TODO: should generate expected layer version resource', () => {
-    // Replaces
-    // https://github.com/serverless/serverless/blob/e78f695004bf292c4163daf9705e5e0c6cbe2592/test/unit/lib/plugins/aws/package/compile/layers/index.test.js#L108-L145
+  it('should generate expected layer version resource', () => {
+    const resourceName = 'layer';
+    const layerResource = cfResources[naming.getLambdaLayerLogicalId(resourceName)];
+    const s3Folder = service.package.artifactDirectoryName;
+    const s3FileName = service.layers[resourceName].package.artifact.split(path.sep).pop();
+
+    expect(layerResource.Type).to.be.equals('AWS::Lambda::LayerVersion');
```

Comment:
in assertions in this test also let's use `to.equal` instead of `to.be.equal`/`to.be.equals`